packages/patterns/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.shared.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "declarationDir": "dist-types"
+  },
+  "references": [],
```

Comment:
Once the i18n package change is merged, we can add a reference here.

Reply:
Should I wait for that package to be merged before merging this? To remove the lint errors?

Reply:
I just went ahead and merged #12198, so just needs a merge/rebase now!